packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2618,32 +2616,36 @@ export class ContainerRuntime
 		await this.pendingStateManager.applyStashedOpsAt(message.sequenceNumber);
 	}
 
-	public process(messageArg: ISequencedDocumentMessage, local: boolean) {
+	/**
+	 * Processes the op.
```

Comment:
Nit: the provided parameter is expressed as a "message", but this comment refers to an "op". If possible, we should bridge this gap. Also, since this is end-user-facing, it would be nice to offer a bit more detail - in this context, what does it mean to "process" the op? When would someone call this?